common/src/main/java/org/opensearch/ml/common/controller/MLModelController.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.controller;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.Getter;
+import org.opensearch.common.xcontent.LoggingDeprecationHandler;
+import org.opensearch.common.xcontent.XContentType;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import org.opensearch.core.common.io.stream.Writeable;
+import org.opensearch.core.xcontent.NamedXContentRegistry;
+import org.opensearch.core.xcontent.ToXContent;
+import org.opensearch.core.xcontent.ToXContentObject;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.core.xcontent.XContentParser;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+import static org.opensearch.core.xcontent.XContentParserUtils.ensureExpectedToken;
+import static org.opensearch.ml.common.utils.StringUtils.getParameterMap;
+
+@Data
+public class MLModelController implements ToXContentObject, Writeable {
+
+    public static final String MODEL_ID_FIELD = "model_id"; // mandatory
+    public static final String USER_RATE_LIMITER_CONFIG = "user_rate_limiter_config";
+
+    @Getter
+    private String modelId;
+    private Map<String, MLRateLimiter> userRateLimiterConfig;
```

Comment:
Add a comment here showing the map key is for users, and value is for RateLimiter?

Reply:
Will do